src/goto-programs/goto_model.h

```diff
@@ -77,7 +77,7 @@ class goto_program_location_numberingt
   /// \param goto_model: will be used to ensure unique numbering of
   ///   goto programs, specifically incrementing its unused_location_number
   ///   member each time a program is re-numbered.
-  goto_program_location_numberingt(goto_modelt &goto_model):
+  explicit goto_program_location_numberingt(goto_modelt &goto_model):
```

Comment:
Wrong commit. (Though in my earlier comment it doubted the overall value of this interface as well.)